pyLibrary/queries/expressions.py

```diff
@@ -1768,7 +1768,7 @@ def to_python(self, not_null=False, boolean=False, many=False):
         return "re.match(" + quote(json2value(self.pattern.json) + "$") + ", " + self.var.to_python() + ")"
 
     def to_sql(self, schema, not_null=False, boolean=False):
-        pattern = schema.db.quote_value(convert.json2value(self.pattern.json))
+        pattern = quote(convert.json2value(self.pattern.json))
```

Comment:
I hope the schema object has a db somewhere; we need it to perform the db-specific quoting of a literal value.  `quote` is for quoting json.